### PR TITLE
Website: List XML integration via dhall-xml-ruby

### PR DESCRIPTION
**Pull request by sjakobi**

### CODE REVIEW

**Comment by sjakobi** [comment]
The logo is wider than the other ones… it doesn't look great! :/

image

File: nix/shared.nix [hunk 1]
```diff
@@ -101,6 +101,12 @@ let
 
       twitter = pkgsNew.callPackage ./twitterLogo.nix { };
 
+      xml =
+        pkgsNew.fetchurl {
+          url    = "https://upload.wikimedia.org/wikipedia/commons/9/9d/Xml_logo.svg";
+          sha256 = "15a9a1ga94l9x1ns759nn8cmvp4x0m634c492ilcv97p1vmx6wj7";
```
Comment (sjakobi) [inline nix/shared.nix]: After some searching, I created the hash from the downloaded file with

    nix-hash --base32 --flat --type sha256 xml.svg

Would there have been an easier way?

**Comment by sjakobi** [comment]
What about this one? https://www.svgrepo.com/svg/106090/xml

File: nix/shared.nix (hunk 1 above)
Comment (Gabriel439) [reply nix/shared.nix]: @sjakobi: `nix-prefetch-url` is the easier way to do this

**Comment by Gabriel439** [comment]
@sjakobi: Yeah, the latter XML logo looks nicer

**Comment by sjakobi** [comment]
Better, I guess:

image

**Pull request merged**